app/Entry.js

```diff
@@ -55,7 +56,7 @@ class Entry extends Component {
           {this.state.initialRouteName === 'true' ? (
             <Stack.Screen
               name='InitialScreen'
-              component={LocationTracking}
+              component={LocationController}
```

Comment:
Since you're making new files, let's call this "Home" or "Main" now

It once was NOT the main page, but now it is.

@kenpugsley any pref on Home or Main?